### ISSUE TITLE
Ice cold is a little too cold

### ISSUE THREAD

**Issue by djkramer123** [d3athrow/vgstation13#36524]
#### Description of issue
Ice makes you very cold with very little amounts

#### Difference between expected and actual behavior
I shouldn't freeze to death ordering a drink with ice in it

#### Steps to reproduce
Go up to your friendly service coworker
Ask for a cup of ice water
They give you 48 units of water 2 units of ice
Take a sip
Fall over from the cold shivering

**Comment by SonixApache** [comment]
https://youtu.be/gRrgeAl-RRU

**Comment by DeityLink** [comment]
Is there one situation where drinks cold enough to give you hypothermia would be desirable? If not then the easy fix is to just give ice the same adj_temp as sodas and making sure cold reagents can't get their adj_temp below that same value.

**Comment by Eneocho** [comment]
the plasmeme on fire but not burning from cold drinks trick no longer works
and I guess being able to partially survive a plasmaflood by chugging soda would be somewhat desirable

**Comment by SonixApache** [comment]
> being able to partially survive a plasmaflood by chugging soda would be somewhat desirable

what

**Comment by Eneocho** [comment]
that's what the behaviour (kinda) was before the whole series of fire PRs, you could (somewhat) tank a plasmafire by chugging soda, since you'd cool down somewhat faster than the 4000K room would heat you
you could not survive indefinitely, but it was enough to dash for a suit or get into a shelter

**Comment by anonymoose-contributor** [comment]
> Is there one situation where drinks cold enough to give you hypothermia would be desirable? 

I can only see this ever being useful for pranks, nothing else. The temperature adjustments (alongside with how it affects drinks, especially those coming from dispensers,) seems to have a possible unintended oversight? Ice isn't the only questionable change either, as the brewing machines for making tea and such also dispense scalding hot milk. Perhaps we should add adjustable temps as a feature to some of those machines

**Comment by Eneocho** [comment]
cold drinks are cold enough that a pAI can kill their owner no problem by dosing some 30u of cola

**Comment by ErrorsWindows** [comment]
Hey bartender, can I get that new ICE-COLD milkshake you've been making?

https://github.com/vgstation-coders/vgstation13/assets/122955989/f6efb0b0-c151-4744-a086-d9129eb7d72a

**Comment by DeityLink** [comment]
> cold drinks are cold enough that a pAI can kill their owner no problem by dosing some 30u of cola

That sounds like a bug, I chugged a can of space cola down the other day and didn't get any hypothermia. Only the "slightly cold" temperature indicator as intended.

> and I guess being able to partially survive a plasmaflood by chugging soda would be somewhat desirable

that's unrelated to my question since soda adj_temp is much higher than ice, it makes you slightly cold by like 1-2°C max, keeping you out of hypothermia range.

**Comment by DeityLink** [comment]
Yeah I just tested locally to use a pAI to give myself 30u of cola and no hypothermia whatsoever here.

**Comment by Eneocho** [comment]
weird

**Comment by adacovsk** [comment]
can be closed@Eneocho
